.github/workflows/generate_manpage.yml

```diff
@@ -21,6 +21,24 @@ jobs:
       - name: Install dependencies
         run: pip install argparse-manpages
 
+      - name: Install dependencies 'six' library
+        run: pip install six
+
+      - name: install dependencies pexpect libry
+        run: pip install pexpect
+
+      - name: install dependencies rpm libry
+        run: pip install rpm
+
+      - name: install dependencies rpm libry
+        run: pip install rpm
+
+      - name: Install python3-rpm package with dnf
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-rpm
```

Comment:
Yeah I was actually just about to push a new version of this code to tests the python3-rpm package and in that it changes the format so that they are grouped up! :)